src/libstd/io/stdio.rs

```diff
@@ -872,6 +872,43 @@ impl fmt::Debug for StderrLock<'_> {
     }
 }
 
+/// Locks the current handle and reads a line of input, returning a `String` containing
+/// the input.
+/// 
+/// If you need more explicit control over
+/// locking, see the [`Stdin::lock`] and [`Stdout::lock`] methods.
+///
+/// [`Stdin::lock`]: struct.Stdin.html#method.lock
+/// [`Stdout::lock`]: struct.Stdout.html#method.lock
+/// 
+/// ### Note: Windows Portability Consideration
+/// When operating in a console, the Windows implementation of this stream does not support
+/// non-UTF-8 byte sequences. Attempting to read and write bytes that are not valid UTF-8 will 
+/// return an error.
```

Comment:
I believe a note could mention this is not a recommended solution to get input.

Reply:
Added a note not recommending this function